website/pages/docs/connect/gateways/index.mdx

```diff
@@ -0,0 +1,64 @@
+---
+layout: docs
+page_title: Gateways
+sidebar_title: Gateways
+description: >-
+  Gateways enable connectivity into, out of, and between Consul service meshes.
+---
+
+# Gateways
+
+## Mesh Gateways
+-> **1.6.0+:** This feature is available in Consul versions 1.6.0 and newer.
+
+Mesh gateways enable routing of service mesh traffic between different Consul datacenters. Those datacenters can reside
+in different clouds or runtime environments where general interconnectivity between all services in all datacenters
+isn't feasible. One scenario where this is useful is when connecting networks with overlapping IP address space.
+
+These gateways operate by sniffing the SNI header out of the mTLS connection and then routing the connection to the
+appropriate destination based on the server name requested. The data within the Connect session is not decrypted by
+the Gateway.
+
+As of Consul 1.8.0, mesh gateways can send all forms of traffic, including gossip and RPC communication between Consul servers. 
+This is enabled by [WAN federation via mesh gateways](/docs/connect/gateways/wan-federation-via-mesh-gateways).
+
+For more information mesh gateways see the [complete documentation](/docs/connect/gateways/mesh-gateway)
+and the [mesh gateway guide](https://learn.hashicorp.com/consul/developer-mesh/connect-gateways).
+
+![Mesh Gateway Architecture](/img/mesh-gateways.png)
+
+## Ingress Gateways
+-> **1.8.0+:** This feature is available in Consul versions 1.8.0 and newer.
+
+Ingress gateways are an entrypoint for outside traffic. They enable potentially unauthenticated ingress traffic from
+services outside the Consul service mesh to services inside the service mesh. 
+
+These gateways allow you to define what services should be exposed, on what port, and by what hostname. You configure
+an ingress gateway by defining a set of listeners that can map to different sets of backing services.
+
+Ingress gateways are tightly integrated with Consul’s L7 configuration and enable dynamic routing of HTTP requests by
+attributes like the request path.
+
+For more information ingress gateways see the [complete documentation](/docs/connect/gateways/ingress-gateway)
+and the [ingress gateway guide](https://learn.hashicorp.com/consul/developer-mesh/ingress-gateways).
+
+![Ingress Gateway Architecture](/img/ingress-gateways.png)
+
+## Terminating Gateways
+-> **1.8.0+:** This feature is available in Consul versions 1.8.0 and newer.
+
+Terminating gateways enable connections from services in the Consul service mesh to services that do not have
+sidecar proxies. These may be services running on legacy infrastructure or managed cloud services running on
```

Comment:
> services that do not have sidecar proxies

Do "connect native services" have sidecars? Is this different from ingress and mesh gateways, or could this say "outside the Consul service mesh" ?

Reply:
That's a good point. Connect native services do not have sidecars. 

Updated the wording to account for that.